src/Package/Impl/ProjectSystem/RProjectLoadHooks.cs

```diff
@@ -72,7 +72,8 @@ internal sealed class RProjectLoadHooks {
             , IRToolsSettings toolsSettings
             , IThreadHandling threadHandling
             , ISurveyNewsService surveyNews
-            , [Import(AllowDefault = true)] IProjectItemDependencyProvider dependencyProvider) {
+            , [Import(AllowDefault = true)] IProjectItemDependencyProvider dependencyProvider
+            , ICoreShell coreShell) {
```

Comment:
if you import `ICoreShell`, can you please replace other usages of `VsAppShell.Current`?